Traceback (most recent call last):
  File "/home/kaustubh/Code3/cov_tester.py", line 97, in <module>
    main(data_path=data_path,batch_size=batch_size,num_epochs=num_epochs,learning_rate=learning_rate,model_path=model_path)
  File "/home/kaustubh/Code3/cov_tester.py", line 75, in main
    test_emb_full_loader = torch.utils.data.DataLoader(torch.utils.data.TensorDataset(X_emb_test_full,Y_test_full), batch_size=1000,
UnboundLocalError: local variable 'X_emb_test_full' referenced before assignment